### PR TITLE
Fix #321

### LINKED ISSUE #321: watson start should not accept empty project
When `options.confirm_new_project` is false, `watson start` gives error when there's no project given.
```
$ watson config options.confirm_new_project false
$ watson start
Error: No project given.
```

However, `watson start` asks if users would like to create empty project when `options.confirm_new_project` is true. This does not look like expected behavior.
```
$ watson config options.confirm_new_project true
$ watson start
Project '' does not exist yet. Create it? [y/N]:
```

### PR DESCRIPTION
**Pull request by hiiwave**
This is to fix #321

### CODE REVIEW

**Comment by hiiwave** [comment]
The tests run well under my environment. I have no idea yet why the CI fail here.

**Comment by hiiwave** [comment]
Agree. Changes applied. 

> Could you add a line in the changelog describing the fix?

Sure. Is this a common practice for PR here? As I didn't do this in #312, #318 and #320 .

**Comment by davidag** [comment]
> Agree. Thanks for the suggestions. Changes applied.

@jmkerr do you have any clue of why `test_report_current` is failing with the changes in this PR? Thanks! :relaxed: 

> 
> > Could you add a line in the changelog describing the fix?
> 
> Sure. Is this a common practice for PR here? As I didn't do this in #312, #318 and #320 .

As far as I have seen, it is common practice here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Stylize prompt to create new project or tag (#310).
 - Aggregate calculates wrong time if used with `--current` (#293)
+- `watson start` now correctly checks if project is empty (#322)
```
Comment (davidag) [inline CHANGELOG.md]: I would be consistent with the rest of the changelog and talk about the `start` command instead of `watson start`.

**Comment by jmkerr** [comment]
> @jmkerr do you have any clue of why `test_report_current` is failing with the changes in this PR? Thanks! ☺️

This is completely unrelated to the content of the commit, but possibly related to the time the test was run between 00:00 and 01:00 UTC. I'll take a look at the problem and open another issue. (I'm sorry.)

**Comment by davidag** [comment]
> This is completely unrelated to the content of the commit, but possibly related to the time the test was run between 00:00 and 01:00 UTC. I'll take a look at the problem and open another issue. (I'm sorry.)

@jmkerr thank you for your help! If there's some way we can help you out, just say it! :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (hiiwave) [reply CHANGELOG.md]: Agree. Changes applied.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Stylize prompt to create new project or tag (#310).
 - Aggregate calculates wrong time if used with `--current` (#293)
+- `start` now correctly checks if project is empty (#322)
```
Comment (jmaupetit) [inline CHANGELOG.md]:
```suggestion
- The `start` command now correctly checks if project is empty (#322)
```

**Pull request merged**